requirements/systemmodel/use-case-diagram.tex

```diff
@@ -0,0 +1,2 @@
+\subsection{Use Case Diagram}
+\includegraphics[width=\linewidth]{./systemmodel/UseCaseDiagram.png}
```

Comment:
1. I think users should be allowed to view task histories as well. However, only administrators should be allowed to edit the task history.
2. What do you mean by manage task?
3. Users should only be allowed to remove their own tasks. Administrators should be allowed to remove any task.
4. Administrators will be the ones to start/stop the central server and the work machine clients.

Reply:
Yea I think i know what you mean. I will try make it better.